vassal-app/src/main/java/VASSAL/launch/BasicModule.java

```diff
@@ -72,6 +72,9 @@
 import VASSAL.tools.SequenceEncoder;
 import VASSAL.tools.menu.MenuManager;
 
+/**
+ * BasicModule holds the
+ */
```

Comment:
Missing words?

Reply:
Ah, you have discovered the EXACT point where the deeper mystery of decode/decode encode/encode brought me up short!